src/partials/messenger.receiver/contact.html

```diff
@@ -10,7 +10,8 @@
 				<dt>Threema ID</dt>
 				<dd>
 					<span class="complex-values">
-						<span>{{ctrl.receiver.id}}</span>
+						<span>{{ ctrl.receiver.id }}</span>
+						<span ng-if="ctrl.receiver.state !== 'ACTIVE'">({{ ctrl.receiver.state }})</span>
```

Comment:
The raw untranslated state is now exposed in the UI. Is this acceptable?

Reply:
Originally that was my intent, yes, but I changed my mind (since we can pull most translation strings from the app).